contracts/feature-tests/composability/promises-features/src/call_sync_bt.rs

```diff
@@ -14,13 +14,14 @@ pub trait BackTransfersFeatureModule {
         token_nonce: u64,
         amount: BigUint,
     ) {
-        self.vault_proxy()
-            .contract(to)
-            .retrieve_funds(token, token_nonce, amount)
-            .execute_on_dest_context::<()>();
+        let (_result, back_transfers) = self.vault_proxy()
```

Comment:
You can write let((), back_transfers) ... .execute_on_dest_context_with_back_transfers();

Reply:
Fixed